src/containerapp/azext_containerapp/containerapp_env_decorator.py

```diff
@@ -149,6 +148,21 @@ def set_up_custom_domain_configuration(self):
                 }
             self.managed_env_def["properties"]["customDomainConfiguration"] = custom_domain
 
+    def set_up_peer_to_peer_encryption(self):
+        is_p2p_encryption_enabled = self.get_argument_p2p_encryption_enabled()
+        is_mtls_enabled = self.get_argument_mtls_enabled()
+        logger.info("mtls: p2p_encryption: %s, mtls: %s", is_p2p_encryption_enabled, is_mtls_enabled)
+
+        if is_p2p_encryption_enabled is not None:
+            safe_set(self.managed_env_def, "properties", "peerTrafficConfiguration", "encryption", "enabled", value=is_p2p_encryption_enabled)
+            if is_p2p_encryption_enabled is False:  # keep the two properties consistent.
+                safe_set(self.managed_env_def, "properties", "peerAuthentication", "mtls", "enabled", value=False)
+
+        if is_mtls_enabled is not None:
```

Comment:
Do we accept `--enable-peer-to-peer-encryption --enable-mtls` with different value?

Reply:
Yes. `--enable-peer-to-peer-encryption true` and `--enable-mtls false` is a valid combination.

Reply:
@LGDoor 
But what if --enable-peer-to-peer-encryption False --enable-mtls?
What is the result?

Reply:
Added validation to fail this combination.